docs/sandbox-permissions.rst

```diff
@@ -165,13 +165,31 @@ filesystem options. For example, ``--filesystem=xdg-documents/path``.
 
 Other filesystem access guidelines include:
 
-- The ``--persist=path`` option can be used to map paths from the user's
-  home directory into the sandbox filesystem.
+- The ``--persist=DIR`` option can be used to map directories from the
+  user's home directory into the sandbox filesystem. This only works if
+  the application has no ``home`` or a broader permission like ``host``
+  that includes ``home``.
+
+  For example, if an application hardcodes the directory ``~/.foo``,
+  without any ``home`` access and no ``--persist`` the directory will be
+  lost from the sandbox once exited. A ``--persist=.foo``
```

Comment:
Guess we could explain its lost because everything is a tmpfs unless mounted over?

Reply:
Added a line for it.